js/tiles.js

```diff
@@ -293,3 +293,11 @@ const door = {
     },
   }),
 };
+outdoor: Object.assign({}, OpenDoors, {
```

Comment:
Please properly add this object to the door class.

Reply:
This needs to be one line higher.

     };

must be at the end of the file.

Reply:
I guess, this is what @abishekvashok wants to say.